ui/src/pages/GameScreen.scss

```diff
@@ -44,13 +44,12 @@ h6.MuiTypography-h6 {
 
 main {
   .logo {
-    margin: 20px 20px 0 20px;
+    margin: 5px 20px 0 20px;
```

Comment:
The Catanatron logo has a slighly thinner black border underneath it so the maritime trading tiles don't get overlapped